lib/destination/dml/merge.go

```diff
@@ -98,17 +99,17 @@ func (m *MergeArgument) GetParts() ([]string, error) {
 		equalitySQLParts = append(equalitySQLParts, equalitySQL)
 	}
 
-	cols := m.Columns.GetEscapedColumnsToUpdate(m.Dialect)
+	columns := m.Columns.GetColumnsToUpdate()
```

Comment:
I renamed this variable so that it would be obvious if I missed a usage of `cols`.